BLSQ/iaso
# IA-3551: Rename permissions on the new visual permissions

Explain what problem this PR is resolving
- Rename permissions on the new visual permissions
Related JIRA tickets : [IA-3551](https://bluesquare.atlassian.net/browse/IA-3551)
## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Rename one permission
- Rename user permissions(geo-limited and admin into all)

## How to test
- Edit or create a user
- Check in the permissions list if the permissions in the ticket have been updated

## Print screen / video
[Screencast from 2024-10-14 11-10-52.webm](https://github.com/user-attachments/assets/99955398-5b3f-48ec-a20a-c83c95fefe6b)


## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3551]: https://bluesquare.atlassian.net/browse/IA-3551?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
